### DISCUSSION TITLE
When will next 2.2.+ version be released?

### DISCUSSION THREAD

**Discussion by mikecollard** [resilience4j/resilience4j#2258] [Q&A]
The latest 2.2.0 release is from 2023 and is not compatible with feign 12.5 or newer.

I see that this has already been resolved and merged in https://github.com/resilience4j/resilience4j/pull/2130 but no new artifact published.

**Comment by RobWin** [comment]
2.3.0 has been released today

**Reply by mikecollard** [reply]
That's great!  Any idea when it will be available via maven central?

**Reply by RobWin** [reply]
I guess its available now
